client/post-editor/media-modal/detail/_style.scss

```diff
@@ -118,36 +118,30 @@
 		@include breakpoint( ">480px" ) {
 			display: block;
 			position: absolute;
-			    top: 8px;
-			    right: 8px;
+				top: 8px;
+				right: 8px;
```

Comment:
👍  I always forget about this [indentation rule](https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/css.md#positioning)